tests/bagatom/test_nodeToXML.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from datetime import datetime
+from lxml import etree
+
+from codalib import bagatom
+
+
+class NodeStub(object):
+    """
+    Test stub that implements the Node model interface.
+    """
+    node_name = "Test Name"
+    node_url = "http://example.com/node"
+    node_path = "/foo/bar/node"
```

Comment:
Is there a reason for these values using double-quotes instead of single-quotes?